<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Date</string>
  </property>
  <widget class="QWidget" name="bday">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="bday_2">
     <item>
      <widget class="QLabel" name="birthdaylbl">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Enter your date of birth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="getBirthday">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Number of days until your birthday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="number">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Georgia&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="openFile">
     <property name="text">
      <string>Open file</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="datesd">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nextDay</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>previousDay</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IsLeap</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WeekNumber</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_date">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>add_new_date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_data">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>change_data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>change_data</sender>
   <signal>released()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
